spec/controllers/questions/consent_controller_spec.rb

```diff
@@ -153,14 +153,16 @@
           email_body = <<~BODY
             Hello Ruthie Rutabaga,
 
-            Thanks for starting your taxes with GetYourRefund - you’re almost there! We will prepare your taxes once we have all of your required information and tax documents.
+            Thanks for starting your taxes with GetYourRefund - you’re almost there! Your case number is #{intake.client_id}. We will prepare your taxes once we have all of your required information and tax documents.
```

Comment:
there's been talk about us hopefully swapping out confirmation number from client_id to something else in the near future and/or next tax year. I think it could be a huge help to have a method on client called confirmation number that looks like this: 

``` 
def confirmation_number
  id
end
```
So that we can reference client.confirmation_number everywhere we need to, and then just swap that value out with the new confimation number and have it update everywhere. I have concerns that if we pass it in as client_id from the intake and id straight from the client sometimes, it will be much harder to track down all the places we're using it later on!

Reply:
Ok! It seems like there's still some back and forth on what to call it/do with this so I'm going to hold off until we nail down the wording.